go/common/grpc/auth/auth_test.go

```diff
@@ -0,0 +1,143 @@
+package auth_test
```

Comment:
need a separate package to avoid circular dependency: this imports `common/grpc/testing` ->`common/grpc`->`common/grpc/auth`. Can move it in a test package under `common/grpc/auth/test` if that's nicer.